src/main/java/mocacong/server/service/FavoriteService.java

```diff
@@ -32,41 +33,38 @@ public class FavoriteService {
     @CacheEvict(key = "#mapId", value = "cafePreviewCache")
     @Transactional
     public FavoriteSaveResponse save(String email, String mapId) {
-        Cafe cafe = cafeRepository.findByMapId(mapId)
-                .orElseThrow(NotFoundCafeException::new);
-        Member member = memberRepository.findByEmail(email)
-                .orElseThrow(NotFoundMemberException::new);
-
+        Cafe cafe = cafeRepository.findByMapId(mapId).orElseThrow(NotFoundCafeException::new);
+        Member member = memberRepository.findByEmail(email).orElseThrow(NotFoundMemberException::new);
```

Comment:
크게 상관없는 부분이긴 하지만, 보통 한줄에 `.`은 하나만 쓰는 컨벤션이 좋기는 해요.
```suggestion
        Cafe cafe = cafeRepository.findByMapId(mapId)
                .orElseThrow(NotFoundCafeException::new);
        Member member = memberRepository.findByEmail(email)
                .orElseThrow(NotFoundMemberException::new);
```

Reply:
넵 유의해서 작성할게요!